mantidproject/mantid
# Fix ties in crystal field fitting

**Description of work.**
This PR adds the peak parameter ties to the function string so that they are passed through to the fit

**Report to:** Tatiana

**To test:**
Do a crystal field fit and set some ties on the peak parameters and check in the parameters workspace that they remain tied.
Contact me or @DanielMurphy22 for the scripts in the issue

Fixes #27748 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
